BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Content/ContentFormView.js

```diff
@@ -68,6 +65,8 @@ class ContentFormView extends mix(AbstractFormView).with(FormViewButtonsMixin)
             this._renderContentActionToolbar($('.content-action-toolbar', this.$el));
         }
         super._renderForm();
+        $('#page-name', this.$el).html($('#oo_content_name', this.$el).val());
```

Comment:
@alavieille : because the value is not known when the initial template is rendered, only when the form is rendered.